docker/ubuntu20/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:20.04
```

Comment:
Can this Dockerfile share any code with the one above?

Reply:
We could use `--build-arg` probably:

```suggestion
FROM ${IMAGE}
```

and run `docker build` with `--build-arg IMAGE="ubuntu:20.04"`



I think we could pass FROM as a `--build-arg` to docker build command

Reply:
Think that's similar to what we do here: https://github.com/Homebrew/homebrew-portable-ruby/blob/master/Dockerfile

Reply:
I'll try that out.

Reply:
Done!